doc/modules/model_evaluation.rst

```diff
@@ -461,6 +461,29 @@ given binary ``y_true`` and ``y_pred``:
     Currently this score function is only defined for binary classification problems, you
     may need to wrap it by yourself if you want to use it for multilabel problems.
 
+    There is no clear consensus on the definition of a balanced accuracy for the
+    multiclass setting. Here are some definitions that can be found in the literature:
+
+    * Normalized class-wise accuracy average as described in [Guyon2015]_
+    * Macro-average recall as described in [Mosley2013]_ and [Kelleher2015]_
+
+    Note that none of these different definitions are currently implemented within
```

Comment:
I would add either here or above that macro-average recall is implemented in sklearn.